lib/animas/animasSimulator.js

```diff
@@ -98,6 +98,12 @@ exports.make = function(config){
           annotate.annotateEvent(event,'animas/out-of-sequence');
         }
 
+        if((currBasal.rate === 0) && (currBasal.deliveryType !== 'suspend')) {
+          // As only the 30 most recent suspend/resume events are stored,
+          // we can't be sure if older basals with 0.0 rate are suspends or not
```

Comment:
Per your comment, would it be possible to tell if we're looking at an "older" basal, so we're not annotating the "newer" basals unnecessarily?